packages/editor/src/components/widgets/collapsible/Collapsible.tsx

```diff
@@ -5,15 +5,21 @@ import { IvyIcons } from '@axonivy/editor-icons';
 import IvyIcon from '../IvyIcon';
 import { MessageText, MessageTextProps } from '../message/Message';
 
-export type CollapsibleProps = MessageTextProps & { label: string; defaultOpen?: boolean; children: ReactNode };
+export type CollapsibleProps = MessageTextProps & { label: string; defaultOpen?: boolean; autoClosable?: boolean; children: ReactNode };
 
-const Collapsible = ({ label, defaultOpen, message, children }: CollapsibleProps) => {
+const Collapsible = ({ label, defaultOpen, message, children, autoClosable }: CollapsibleProps) => {
   const [open, setOpen] = useState(defaultOpen ?? false);
   useEffect(() => {
-    if (defaultOpen) {
-      setOpen(defaultOpen);
+    if (autoClosable) {
+      if (defaultOpen !== undefined) {
+        setOpen(defaultOpen);
+      }
+    } else {
+      if (defaultOpen) {
+        setOpen(defaultOpen);
+      }
```

Comment:
so is this required, that this stays open?
If not, I would definitely vote for keeping it simple and just close it...

Reply:
I've adjusted the component with the 'autoClosable' feature so that the Collapsible continues to work the same as before everywhere, and in special cases like the browsers, the open/close function can be ensured. However, I believe we'll discuss this issue briefly tomorrow, and then we'll have clarity on how the helper text should behave. For now, I think I'll leave it like this, and if it proves to be too much, I can revert to the simpler version without the automatic closing support.